library/general/allowedrepos/template.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: k8sallowedrepos
   annotations:
     metadata.gatekeeper.sh/title: "Allowed Repositories"
+    version: 1.0.0
```

Comment:
@maxsmythe do you think we need `metadata.gatekeeper.sh/` prefix here?

Reply:
I think fully qualified names make sense here.

Reply:
updated to a fully qualified name.